vocode/streaming/action/base_action.py

```diff
@@ -13,6 +13,7 @@
 class BaseAction(Generic[ParametersType, ResponseType]):
     description: str = ""
     action_type: str = ActionType.BASE.value
+    quiet: bool = False
```

Comment:
should this be a constructor argument instead? so it's easier to use in the `ActionFactory`, i.e.

```
return MyAction(quiet=True)
```

Reply:
Moved to the constructor :+1: